docs/schema.md

```diff
@@ -564,6 +575,8 @@ The defined ecosystems are:
 | `AlmaLinux` | AlmaLinux package ecosystem; the `name` is the name of the source package. The ecosystem string might optionally have a `:<RELEASE>` suffix to scope the package to a particular AlmaLinux release. `<RELEASE>` is a numeric version.
 | `Bitnami` | Bitnami package ecosystem; the `name` is the name of the affected component. |
 | `Photon OS` | The Photon OS package ecosystem; the `name` is the name of the RPM package. The ecosystem string must have a `:<RELEASE-NUMBER>` suffix to scope the package to a particular Photon OS release. Eg `Photon OS:3.0`. |
+| `CRAN` | The biological R package ecosystem. The `name` is an R package name. |
```

Comment:
Are there any additional subtleties or rules around the package names? E.g. PyPI has one on requiring names to be "normalized" since there's infinite ways to refer to the exact same package.

Reply:
From [Writing R Extensions](https://cran.r-project.org/doc/manuals/R-exts.html#The-DESCRIPTION-file):

> The mandatory ‘Package’ field gives the name of the package. This should contain only (ASCII) letters, numbers and dot, have at least two characters and start with a letter and not end in a dot

Which seems straightforward, the only other subtlety I know of that is somewhat different than PyPI's normalization is that packages are case-sensitive meaning (foo and Foo are two separate packages)

Reply:
yep. duplicate names are not allowed on CRAN, the unique package name is defined by the Package entry in the DESCRIPTION file (DCF).